src/directLine.ts

```diff
@@ -308,7 +308,8 @@ export class DirectLine implements IBotConnection {
     private streamUrl: string;
     public referenceGrammarId: string;
 
-    private pollingInterval: number = 1000;
+    private pollingInterval: number = 1000; //ms
+    private readonly POLLING_INTERVAL_LOWER_BOUND: number = 200; //ms
```

Comment:
This should either be a static or pulled out as a const outside the class definition

Reply:
I'm a little confused. `const` is a runtime check, `readonly` is a compile time check. What am I missing?

Reply:
`const` values cannot be changed, all other definitions can - it's super simple to bypass TS checks at build time. Also, naming - private members should not be upper cased. Static members can be.

Reply:
ok. I'm newish to TypeScript, but a quick learner!